src/irmin-pack/irmin_pack.ml

```diff
@@ -428,7 +428,7 @@ struct
                     let contents : 'a Contents.t = contents in
                     let node : 'a Node.t = (contents, node) in
                     let commit : 'a Commit.t = (node, commit) in
-                    f contents node commit)))
+                    f contents node commit ) ) )
```

Comment:
In some places, there are no spaces _between_ multiline parens, with this being an exception. i.e. we have some instances of `␣))` and some of `␣)␣)`. Is there a reason for that?

Reply:
Ys having `␣))` instead of `␣)␣)` is a bug, we have fixed lots of them, the remaining ones are either tricky or difficult to fix due to all the customization we have on the Format boxes, but we aim to fix them as well